L1Trigger/L1THGCal/src/concentrator/HGCalConcentratorAutoEncoderImpl.cc

```diff
@@ -173,6 +174,13 @@ void HGCalConcentratorAutoEncoderImpl::select(unsigned nLinks,
   }
 
   if (modSum > 0) {
+    //Use a bit shift normalization like will be implemented in ECON, rather than floating point sum
+    //Normalizes to the MSB value of the module sum
+    if (bitShiftNormalization_){
+      int msb = int(log2(modSum));
+      modSum = pow(2,msb);
```

Comment:
Would be much more efficient to use `1<<msb` instead of `pow(2,msb)`

Reply:
Because the value used for filling the AE input and the modSum is mipPt (rather than taking a quantized version of the transverse ADC), it is possible to have a modSum < 1, and a negative msb.  The negative value of msb is handled by `pow(2,msb)` but now `1<<msb`.